### PR TITLE
Re-generate the sample kubernetes manifests from the Helm chart 2.28.13

### PR DESCRIPTION
**Pull request by L3n41c**
### What does this PR do?

Regenerate the sample kubernetes manifests from the latest version of the Helm chart.

### Motivation

Keep those kubernetes manifests up-to-date

### Preview

https://docs-staging.datadoghq.com/<BRANCH_NAME>/<PATH>

### Additional Notes

This PR has been fully generated by the execution of the [`generate.sh` script](https://github.com/DataDog/documentation/blob/master/static/resources/yaml/generate.sh).

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: static/resources/yaml/datadog-agent-windows-apm.yaml [hunk 1]
```diff
@@ -218,9 +244,9 @@ spec:
         - hostPath:
             path: \\.\pipe\docker_engine
           name: runtimesocket
-        # If the CRI is not provided, try to mount the default containerd pipe.
-        # By default, datadog.dockerOrCriSocketPath mounts the Docker pipe.
-        # With this additional hostPath, both are mounted.
+        # if the CRI or is not provived we should try to mount the default containerd pipe.
```
Comment (maycmlee) [inline static/resources/yaml/datadog-agent-windows-apm.yaml:247]:
```suggestion
        # if the CRI is not provided, try to mount the default containerd pipe.
```

File: static/resources/yaml/datadog-agent-windows-apm.yaml [hunk 2]
```diff
@@ -218,9 +244,9 @@ spec:
...
+        # by default "datadog.dockerOrCriSocketPath" mount the docker pipe.
```
Comment (maycmlee) [inline static/resources/yaml/datadog-agent-windows-apm.yaml:248]:
```suggestion
        # By default, "datadog.dockerOrCriSocketPath" mounts the Docker pipe.
```

File: static/resources/yaml/datadog-agent-windows-apm.yaml [hunk 3]
```diff
@@ -218,9 +244,9 @@ spec:
...
+        # so with this additional hostPath, by default both are mounted.
```
Comment (maycmlee) [inline static/resources/yaml/datadog-agent-windows-apm.yaml:249]:
```suggestion
        # So with this additional hostPath, by default, both are mounted.
```

File: static/resources/yaml/datadog-agent-windows-logs-apm.yaml [hunk 4]
```diff
@@ -241,9 +267,9 @@ spec:
         - hostPath:
             path: \\.\pipe\docker_engine
           name: runtimesocket
-        # If the CRI is not provided, try to mount the default containerd pipe.
-        # By default, datadog.dockerOrCriSocketPath mounts the Docker pipe.
-        # With this additional hostPath, both are mounted.
+        # if the CRI or is not provived we should try to mount the default containerd pipe.
```
Comment (maycmlee) [inline static/resources/yaml/datadog-agent-windows-logs-apm.yaml:270]:
```suggestion
        # If the CRI is not provided, try to mount the default containerd pipe.
```

File: static/resources/yaml/datadog-agent-windows-logs-apm.yaml [hunk 5]
```diff
@@ -241,9 +267,9 @@ spec:
...
+        # by default "datadog.dockerOrCriSocketPath" mount the docker pipe.
```
Comment (maycmlee) [inline static/resources/yaml/datadog-agent-windows-logs-apm.yaml:271]:
```suggestion
        # By default, "datadog.dockerOrCriSocketPath" mounts the Docker pipe.
```

File: static/resources/yaml/datadog-agent-windows-logs-apm.yaml [hunk 6]
```diff
@@ -241,9 +267,9 @@ spec:
...
+        # so with this additional hostPath, by default both are mounted.
```
Comment (maycmlee) [inline static/resources/yaml/datadog-agent-windows-logs-apm.yaml:272]:
```suggestion
        # So with this additional hostPath, by default, both are mounted.
```

File: static/resources/yaml/datadog-agent-windows-logs.yaml [hunk 7]
```diff
@@ -191,9 +213,9 @@ spec:
         - hostPath:
             path: \\.\pipe\docker_engine
           name: runtimesocket
-        # If the CRI is not provided, try to mount the default containerd pipe.
-        # By default, datadog.dockerOrCriSocketPath mounts the Docker pipe.
-        # With this additional hostPath, both are mounted.
+        # if the CRI or is not provived we should try to mount the default containerd pipe.
```
Comment (maycmlee) [inline static/resources/yaml/datadog-agent-windows-logs.yaml:216]:
```suggestion
        # If the CRI is not provided, try to mount the default containerd pipe.
```

File: static/resources/yaml/datadog-agent-windows-logs.yaml [hunk 8]
```diff
@@ -191,9 +213,9 @@ spec:
...
+        # by default "datadog.dockerOrCriSocketPath" mount the docker pipe.
```
Comment (maycmlee) [inline static/resources/yaml/datadog-agent-windows-logs.yaml:217]:
```suggestion
        # By default, "datadog.dockerOrCriSocketPath" mounts the Docker pipe.
```

File: static/resources/yaml/datadog-agent-windows-logs.yaml [hunk 9]
```diff
@@ -191,9 +213,9 @@ spec:
...
+        # so with this additional hostPath, by default both are mounted.
```
Comment (maycmlee) [inline static/resources/yaml/datadog-agent-windows-logs.yaml:218]:
```suggestion
        # So with this additional hostPath, by default, both are mounted.
```

File: static/resources/yaml/datadog-agent-windows-vanilla.yaml [hunk 10]
```diff
@@ -170,9 +192,9 @@ spec:
         - hostPath:
             path: \\.\pipe\docker_engine
           name: runtimesocket
-        # If the CRI is not provided, try to mount the default containerd pipe.
-        # By default, datadog.dockerOrCriSocketPath mounts the Docker pipe.
-        # With this additional hostPath, both are mounted.
+        # if the CRI or is not provived we should try to mount the default containerd pipe.
```
Comment (maycmlee) [inline static/resources/yaml/datadog-agent-windows-vanilla.yaml:195]:
```suggestion
        # if the CRI is not provided, try to mount the default containerd pipe.
```

File: static/resources/yaml/datadog-agent-windows-vanilla.yaml [hunk 11]
```diff
@@ -170,9 +192,9 @@ spec:
...
+        # by default "datadog.dockerOrCriSocketPath" mount the docker pipe.
```
Comment (maycmlee) [inline static/resources/yaml/datadog-agent-windows-vanilla.yaml:196]:
```suggestion
        # By default, "datadog.dockerOrCriSocketPath" mounts the Docker pipe.
```

File: static/resources/yaml/datadog-agent-windows-vanilla.yaml [hunk 12]
```diff
@@ -170,9 +192,9 @@ spec:
...
+        # so with this additional hostPath, by default both are mounted.
```
Comment (maycmlee) [inline static/resources/yaml/datadog-agent-windows-vanilla.yaml:197]:
```suggestion
        # So with this additional hostPath, by default, both are mounted.
```

**Review by maycmlee — CHANGES_REQUESTED** [review]

**Comment by L3n41c** [comment]
In order to avoid having the fixes being reverted at the next manifest generation, I’ve retrofitted the suggested changes upstream in DataDog/helm-charts#505.

File: static/resources/yaml/datadog-agent-windows-apm.yaml [hunk 13]
```diff
@@ -218,9 +244,9 @@ spec:
         - hostPath:
             path: \\.\pipe\docker_engine
           name: runtimesocket
-        # If the CRI is not provided, try to mount the default containerd pipe.
-        # By default, datadog.dockerOrCriSocketPath mounts the Docker pipe.
-        # With this additional hostPath, both are mounted.
+        # if the CRI is not provided, try to mount the default containerd pipe.
```
Comment (maycmlee) [inline static/resources/yaml/datadog-agent-windows-apm.yaml:247]:
```suggestion
        # If the CRI is not provided, try to mount the default containerd pipe.
```

File: static/resources/yaml/datadog-agent-windows-vanilla.yaml [hunk 14]
```diff
@@ -170,9 +192,9 @@ spec:
         - hostPath:
             path: \\.\pipe\docker_engine
           name: runtimesocket
-        # If the CRI is not provided, try to mount the default containerd pipe.
-        # By default, datadog.dockerOrCriSocketPath mounts the Docker pipe.
-        # With this additional hostPath, both are mounted.
+        # if the CRI is not provided, try to mount the default containerd pipe.
```
Comment (maycmlee) [inline static/resources/yaml/datadog-agent-windows-vanilla.yaml:195]:
```suggestion
        # If the CRI is not provided, try to mount the default containerd pipe.
```

**Review by maycmlee — APPROVED** [review]

**Pull request merged**